spec/models/user_spec.rb

```diff
@@ -72,4 +72,12 @@
       expect(user.valid_sso_auth_token?(sso_auth_token)).to eq false
     end
   end
+
+  describe 'access token' do
+    it 'creates a single access token upon user creation' do
+      new_user = create(:user)
+      token_count = AccessToken.where(owner_id: new_user.id).count
```

Comment:
```suggestion
      token_count = AccessToken.where(owner: new_user).count
```

`owner` attribute is a polymorphic column. so we should pass both `owner_id` and `owner_type`, or pass the object and rails takes care of it for us.